.circleci/config.yml

```diff
@@ -295,7 +305,7 @@ jobs:
 
   run-lint:
     docker:
-      - image: mbgl/android-ndk-r21b:latest
+      - image: mbgl/android-ndk-r21:latest
```

Comment:
why do we use r21? I think we should use `21e`, which is the latest LTS and align across different teams.